File: Java OOP Concept/Encapsulation.java
class BankAccount {
    private double balance = 0;

    public void deposit(double amount) {
        balance += amount;
    }

    public double getBalance() {
        return balance;
    }
}

public class Encapsulation {
    public static void main(String[] args) {
        BankAccount account = new BankAccount();
        account.deposit(100);
        System.out.println("Balance: " + account.getBalance());
    }
}

File: Flutter or Dart OOP Concept/Flutter OOP/Encapsulation.dart
class BankAccount {
	double _balance = 0;

	void deposit(double amount) {
		_balance += amount;
	}

	double get balance => _balance;
}

void main() {
	BankAccount account = BankAccount();
	account.deposit(100);
	print('Balance: [32m${account.balance}[0m');
}

